extensions/notification/utils/config.js

```diff
@@ -0,0 +1,89 @@
+const constants = require('../constants');
+
+/**
+ * Transforms array of configurations to a Map
+ * @param  {Object[]} configs - Array of bucket notification configurations
+ * @return {Map} Map of bucket notification configurations
+ */
+function transform(configs) {
+    const configMap = new Map();
+    configs.forEach(config => {
+        if (config.bucket && config.notificationConfiguration) {
+            const bucket = config.bucket;
+            const conf = config.notificationConfiguration.queueConfigurations;
```

Comment:
similarly, `conf` seems a too generic name, `queueConf` could be more precise

Reply:
You resolved this but don't see a change, did you intend to change it but forgot?